company_linkage/parat_data_dag.py

```diff
@@ -0,0 +1,374 @@
+        "intermediate_dataset": intermediate_dataset,
+        "initial_dataset": initial_dataset,
+        "backups_dataset": backups_dataset,
+    },
+)
+with dag:
+
+    clear_tmp_dir = GCSDeleteObjectsOperator(
+        task_id="clear_tmp_dir",
+        bucket_name=DATA_BUCKET,
+        prefix=tmp_dir
+    )
+
+    # combine all the airtable tables into joined tables
+
+    start = DummyOperator(task_id="starting")
+
+    join_tables = []
+    for table in ["alias", "grid", "ids", "linkedin", "market", "organizations", "parent", "permid"]:
+
+        # Grab all the data and write it to unseen_en_corpus
+        join_table = BigQueryInsertJobOperator(
+            task_id=f"join_{table}",
+            configuration={
+                "query": {
+                    "query": f"select distinct * from {initial_dataset}.{table}_preannotation UNION DISTINCT "
+                             f"select distinct * from {initial_dataset}.{table}_validate",
+                    "useLegacySql": False,
+                    "destinationTable": {
+                        "projectId": PROJECT_ID,
+                        "datasetId": initial_dataset,
+                        "tableId": f"{table}_joined"
+                    },
+                    "allowLargeResults": True,
+                    "createDisposition": "CREATE_IF_NEEDED",
+                    "writeDisposition": "WRITE_TRUNCATE"
+                }
+            }
+        )
+        join_tables.append(join_table)
+
+    # Do initial query sequence
+
+    start_initial_tables = DummyOperator(task_id="start_initial_tables")
+
+    wait_for_initial_tables = DummyOperator(task_id="wait_for_initial_tables")
+
+    seq_path_prefix = f"{os.environ.get('DAGS_FOLDER')}/sequences/parat/"
+    initial_query_sequence = "initial_data.csv"
+
+    curr = start_initial_tables
+    for line in open(seq_path_prefix + initial_query_sequence).readlines():
+        dataset, table = line.split(",")
+        table_name = f"{dataset}.{table.strip()}"
+        next_tab = BigQueryInsertJobOperator(
+            task_id=f"create_{table_name}",
+            configuration={
+                "query": {
+                    "query": "{% include '" + f"{sql_dir}/{table.strip()}.sql" + "' %}",
+                    "useLegacySql": False,
+                    "destinationTable": {
+                        "projectId": PROJECT_ID,
+                        "datasetId": dataset,
+                        "tableId": table
+                    },
+                    "allowLargeResults": True,
+                    "createDisposition": "CREATE_IF_NEEDED",
+                    "writeDisposition": "WRITE_TRUNCATE"
+                }
+            },
+        )
+        curr >> next_tab
+        curr = next_tab
+    curr >> wait_for_initial_tables
+
+    # run aggregate_organizations python and load to GCS
+    aggregated_table = "aggregated_organizations"
+
+    aggregate_organizations = PythonOperator(
```

Comment:
This doesn't have any significant disc or memory requirements, right? If it does we should move it to a GKE operator since we'll have to do that anyway when we move this dag to cloud composer 2. If not it's ok to leave it as a python operator.

Reply:
Nope, extremely limited disk and memory requirements; that's why it's in a Python operator when most other things were using GKE. I can switch it over if you want though.

Reply:
No that's fine, just double-checking